### ISSUE TITLE
Operator: Softmax

### ISSUE THREAD

**Issue by ezyang** [onnx/onnx#27]
Requested in https://github.com/pytorch/pytorch/issues/2747

**Comment by ezyang** [comment]
Already supported.
